_sass/_07_layout.scss

```diff
@@ -48,6 +48,10 @@ body.video cite { color: #fff; }
 #navigation {
     -webkit-box-shadow: 0 2px 2px 0 rgba(0,0,0,.2);
     box-shadow: 0 2px 3px 0 rgba(0,0,0,.2);
+
+    [class^='icon-']:before, [class*=' icon-']:before {
```

Comment:
Thanks to @TWiStErRob for this one. Plucked from https://github.com/TWiStErRob/twisterrob.github.io/blob/0ba9f2696ba3ec1105166927f0e1bdd4e3dfb2bd/_sass/_07_layout.scss#L60-L63